gui_qt/games_registry_actions.py

```
@@ -42,12 +48,58 @@ class RegistryActionResult:
+        plan,
+        import_md=import_md,
+        discover=discover,
+        render_md=True if render_md else None,
+        create_directory=create_directory,
+        persist_workspace_root=persist_workspace_root,
+        refresh_new=refresh_new,
+    )
+    try:
+        result: WorkspaceSetupResult = apply_workspace_setup(plan, options)
+    except (OSError, ValueError) as exc:
+        return RegistryActionResult(False, f"接入工作区失败：{exc}")
```

Ruff (0.15.21)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)